tmk_core/protocol/xt_interrupt.c

```diff
@@ -37,6 +37,7 @@ POSSIBILITY OF SUCH DAMAGE.
 */
 
 #include <stdbool.h>
+#include <stdio.h>
```

Comment:
```suggestion
```

This should be removed, and the print.h include replaced with debug.h (I can't add a suggestion there as it's outside of this hunk).